serialization/README.md

```diff
@@ -69,12 +71,14 @@ Examples of how to serialize the following cases:
 The following list begins with base examples and sequentially adds expositional elements and relationships step-by-step:
 
 - An initial set of vulnerability elements
-- Adding vulnerability elements with security external reference types including FIXME
-- Adding hasAssociatedVulnerability relationship between a vulnerability element and a software profile element
-- Adding multiple hasAssessmentFor relationships for vulnerability assessment relationships between vulnerability element and package element for VEX, CVSS, etc. to communicate, e.g.,
-  - changes to a vulnerability element’s status affecting a specific package element using VEX (Vulnerability Exploitability eXchange)  (see the serialized examples listed in Syntax under each SPDX-3-MODEL/model/Security Class definition)
-  - how a vulnerability element may be fixed for a particular software package element
-  - results of a vulnerability scan or audit
-- Adding foundBy, publishedBy, reportedBy relationships between an agent element and a vulnerability element
-
-### Build Profile use cases to be added here
+- Adding vulnerability elements with security external reference types including *FIXME*
```

Comment:
*FIXME* still needs to be resolved.

Reply:
Remove for now

Reply:
I have now put it as:

> - Adding vulnerability elements with security external reference types
  including `securityFix`, `vulnerabilityDisclosureReport`,
  and `vulnerabilityExploitabilityAssessment`

will that work?